### DISCUSSION TITLE
Bitcoin Split Payment for % commission Wordpress Woocomerce

### DISCUSSION THREAD

**Discussion by dela15** [btcpayserver/btcpayserver#2260] [Technical Support]
Hi everyone,

I'm new to BTCpay. I and I was wondering if there were a way to do split payment with the BTCpay woocomerce Bitcoin gateway. Like when I vendor sell something that a % commission goes to Market Admin. Same for an affiliate product.

Thank you

**Comment by pavlenex** [comment]
Hi, no, out of the box that's not possible. If you're technical and have time to experiment, you can experimenting with https://docs.btcpayserver.org/Transmuter/ or if you're a developer you may look into our new API https://docs.btcpayserver.org/API/Greenfield/v1/ (in ~2 weeks we'll have the wallet API).

**Reply by dela15** [reply]
Alright thank you @pavlenex , I'm not a developer. So I guess that for the moment I will have to make the transfert manually to the vendor.
